recipes.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+return [
```

Comment:
we can extend this config for adding gitignors and specific package post-install info